src/app/pages/task-calendar/cron/cron-form/cron-form.component.ts

```diff
@@ -64,8 +64,9 @@ export class CronFormComponent {
            type: 'combobox',
            name: 'cron_user',
            placeholder: T('Run As User'),
-           tooltip: T('Choose a user account to run the <b>Command</b>. The\
-             user must have permission to run the <b>Command</b>.'),
+	   tooltip: T('Select a user account to run the <i>Command</i>\
```

Comment:
"specified" is redundant, and I don't think we need to refer to "command" as a field name.  Also, the part about permission is ...short.  So:

Select a user account to run the command. The user must have permissions allowing them to run the 
command.

Reply:
done